src/app/components/lineage-graph/lineage-graph-heigth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+    providedIn: "root",
+})
+export class LineageGraphHeigthService {
+    private componentsPaddings = 90;
+    private _datasetViewHeaderHeight = 0;
+    public setDatasetViewHeaderHeight(value: number): number {
+        return (this._datasetViewHeaderHeight = value);
+    }
+    public get datasetViewHeaderHeight(): number {
+        return this._datasetViewHeaderHeight;
+    }
+
+    private _appHeaderHeight = 0;
+    public setAppHeaderHeight(value: number): number {
+        return (this._appHeaderHeight = value);
+    }
+    public get appHeaderHeight(): number {
+        return this._appHeaderHeight;
+    }
+
+    private _datasetViewMenuHeight = 0;
+    public setDatasetViewMenuHeight(value: number): number {
+        return (this._datasetViewMenuHeight = value);
+    }
+    public get datasetViewMEnuHeight(): number {
+        return this._datasetViewMenuHeight;
+    }
+
+    public get lineageGraphHeigth(): number {
```

Comment:
With a more general point of view, this is the "dataset tab height" - basically, full window - app header - menu - dataset-header. It will be the same value for other tabs, so no need to call it this concrete.

Reply:
Renamed